readme.md

```diff
@@ -14,6 +14,8 @@ Intended to make it easier to create and maintain Awesome lists.
 
 Includes a bunch of [general Markdown rules](https://github.com/sindresorhus/awesome-lint/blob/master/config.js) and some [Awesome specific rules](https://github.com/sindresorhus/awesome-lint/tree/master/rules).
 
+Supports to enable, disable, and ignore rules by special comments, with the help of [remark-message-control](https://github.com/remarkjs/remark-message-control).
```

Comment:
Include an example of how to disable an Awesome rule.

Reply:
And I think you should explain the difference between ignoring and disabling a rule.

Reply:
I think this could be a separate section

Reply:
I think documents of `remark-message-control` is enough. Maybe [markers section](https://github.com/remarkjs/remark-message-control#markers) is clearer.

Reply:
> I think documents of remark-message-control is enough.

I disagree.

> Maybe markers section is clearer.

Yes, that's what we should link to, but I still think we need better docs here too.

Reply:
I updated the link and the sentence. As I have no ideas to write a better doc, can we merge this work first and add docs later?

In fact, `remark-message-control` has used different examples to show how to write its marks. Even though the difference between ignoring and disabling a rule is not very obvious, reading carefully can still find out that. Let clever users try this feature and give feedbacks.

Reply:
> can we merge this work first and add docs later?

In open source, "later" means "never".

> Let clever users try this feature and give feedbacks.

I don't accept making this accessible just for *clever users*.

Reply:
I am not escaping from writing docs. But need some tips as directions. If leave me alone, it will become much similar with `remark-message-control`'s.